honestcash-v2/src/app/shared/components/avatar/avatar.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+             selector: 'app-avatar',
+             template: `<img class="avatar" [ngClass]="{'large': large, 'small': !large}" [src]="src"/>`,
+             styles: [
+               '.avatar {\n' +
+               '  border-style: solid;\n' +
+               '  border-color: #e7e7e7;\n' +
+               '  border-width: 1px;\n' +
+               '  border-radius: 50%;\n' +
+               '}' +
+               '.avatar.large {\n' +
+               '  height: 45px;\n' +
+               '  width: 45px;\n' +
+               '}' +
+               '.avatar.small {\n' +
+               '  height: 34px;\n' +
+               '  width: 34px;\n' +
+               '}'
+             ]
+           })
+export class AvatarComponent {
+  private _src = '';
+  @Input() large = false;
```

Comment:
missing visibility property.

Reply:
what do you mean?

Reply:
`public` is missing.

Reply:
you mean access modifier? so by default all @Input values are set from outside meaning that all of them are public. but I'll add this unnecessary public to both the large, the getter and setters